package.json

```diff
@@ -3,7 +3,8 @@
     "eslint": "eslint .",
     "eslint-fix": "eslint --fix .",
     "typecheck": "tsc --project jsconfig.json --maxNodeModuleJsDepth 0 --noEmit",
-    "mdlint": "markdownlint '**/*.md' --config .mdl.json --ignore '**/node_modules/**' --ignore 'code-of-conduct.md'"
+    "mdlint": "markdownlint '**/*.md' --config .mdl.json --ignore '**/node_modules/**' --ignore 'code-of-conduct.md'",
+    "rebuild-package-lock": "find . -name package.json -a ! -path '*/node_modules/*' -execdir pwd \\;  -execdir npm install --save \\;"
```

Comment:
To do only the `package-lock.json` files you could do `npm install --package-lock-only` and skip the actual install? Or perhaps rename to `install-all`/`install-src-tree`? Or even add both.

Reply:
install-all makes more sense :)

Reply:
Nice -- possible also to update the commit message to match?